src/OrleansServiceBus/Providers/Streams/EventHub/EventHubCheckpointer.cs

```diff
@@ -22,16 +19,16 @@ internal class EventHubPartitionCheckpoint
         private Task inProgressSave;
         private DateTime? throttleSavesUntilUtc;
 
-        public bool Exists { get { return entity != null && entity.Offset != EventHubConsumerGroup.StartOfStream; } }
+        public bool Exists => entity != null && entity.Offset != EventHubConsumerGroup.StartOfStream;
```

Comment:
@jdom we are using VSO hosted build agent for our project and it builds C# 6 syntax without any problems. Can you point to some page with more info on this ?